src/components/homepage/HomePage.js

```diff
@@ -90,7 +90,7 @@ export function HomeContent() {
       />
       <Container className={classes.container}>
         <img src={Logo} width={"50%"}></img>
-        <Title className={classes.title}>a concept to incorporate operational experience into software projects</Title>
+        <Title className={classes.title}>Building an all-Open Source community cloud to learn stuff and make <Link to="/about">things</a> better</Title>
```

Comment:
You need to close the <Link> tag with a </Link> not </a> :) old habbits

Reply:
HA! I literally hand-typed your fix from the other comment and thus introduced an error. :grin: 

```suggestion
        <Title className={classes.title}>Building an all-Open Source community cloud to learn stuff and make <Link to="/about">things</link> better</Title>
```